spec/models/activity_spec.rb

```diff
@@ -85,5 +85,30 @@
       it { should validate_presence_of(:finance) }
       it { should validate_presence_of(:tied_status) }
     end
+
+    context "checks dates are within accepted parameters" do
```

Comment:
I've come to realise that eveyone or every team has a different take on contexts, it, describe, etc, but is this a context? Does this still read well without the nesting?

When I look at the rest of this spec I feel like the date tests would all be in the `validations` describe block with each a context?

That said the `spec/models/transaction_spec.rb` seems to take this approach – I guess I care more that we are consistent accross them all more than which we use?

Reply:
Agree with this!

Reply:
Hmm, so that raises the questions which do we prefer and do we refactor to make it so?

Reply:
If we decide to refactor then that's one for another ticket (IMHO)